### DISCUSSION TITLE
PIXI.js is Request

### DISCUSSION THREAD

**Discussion by RobinYang11** [vercel/next.js#28068] [Ideas]
### Describe the feature you'd like to request

please help me !
https://www.pixijs.com/

### Describe the solution you'd like

i don't know how to support pixi.js
![image](https://user-images.githubusercontent.com/20513480/129299513-02dfc1eb-dbf1-4a70-90db-6c2ee264dea6.png)

### Describe alternatives you've considered

![image](https://user-images.githubusercontent.com/20513480/129299524-515b08a0-c080-4c58-9b79-dfc77b59eb7b.png)

**Comment by ByamB4** [comment]
Are you trying to use **pixi.js** to your next.js project ?
